.github/workflows/linux-aarch64.yml

```diff
@@ -20,7 +20,10 @@ jobs:
       runner: linux.arm64.2xlarge
       test-matrix: |
         { include: [
-          { config: "default", shard: 1, num_shards: 1, runner: "linux.arm64.2xlarge" },
+          { config: "default", shard: 1, num_shards: 4, runner: "linux.arm64.2xlarge" },
+          { config: "default", shard: 2, num_shards: 4, runner: "linux.arm64.2xlarge" },
+          { config: "default", shard: 3, num_shards: 4, runner: "linux.arm64.2xlarge" },
+          { config: "default", shard: 4, num_shards: 4, runner: "linux.arm64.2xlarge" },
```

Comment:
🤦 Note to self: be a more thorough code reviewer... This just runs the same tests 1 times over...